odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -2,6 +2,22 @@
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+_TEXT_WARNINGS = {
+    ".xml": {
+        r".*active.*toggle_button":
```

Comment:
Looks like a very rare case. Odoo 12 has just two examples (and those are in opposite order): https://odoo-source.com/?q=toggle_button.*active&i=nope&files=&excludeFiles=po%24%7Cpot%24%7Cyml%24%7Cyaml%24%7Ccss%24%7C%2Fstatic%2Flib%2F&repos=12

But should be fine to keep it